### DISCUSSION TITLE
Neorg files (`.norg`) previews?

### DISCUSSION THREAD

**Discussion by xfzv** [sxyazi/yazi#1351] [1. Q&A]
I'm using [Neorg](https://github.com/nvim-neorg/neorg) for note taking. It would be nice if `.norg` files could be previewed with syntax highlighting, just like markdown files can be previewed with [Reledia/glow.yazi](https://github.com/Reledia/glow.yazi).

**Comment by AnirudhG07** [comment]
I couldn't find a pre existing tool to view .norg files. but there is a way of converting it to markdown and viewing it using glow, although it defeats the purpose but yeah, thats one way.
